dasharo-security/secure-boot.robot

```diff
@@ -285,6 +285,58 @@ SBO009.002 Check automatic certificate provisioning KEK certificate
     ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
     Reset To Default Secure Boot Keys    ${advanced_menu}
 
+SBO010.001 Enroll certificates using sbctl
+    [Documentation]    This test installs sbctl and verifies that it is possible
+    ...    to use it to enroll certificates.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO010.001 not supported
+    Power On
+    # Erase Secure Boot Keys
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    Enable Secure Boot    ${sb_menu}
+    Save Changes
+    Reenter Menu
+    ${sb_menu}=    Get Secure Boot Menu Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    # Install sbctl
+    Execute Linux Command    apt install -y git util-linux binutils golang-go asciidoc make    120
+    Execute Linux Command    [ ! -d "sbctl" ] && git clone https://github.com/foxboron/sbctl.git
+    ${new_prompt}=    Get Linux Prompt In Directory    sbctl    ${DEVICE_UBUNTU_ROOT_PROMPT}
```

Comment:
@PLangowski this new keyword is only needed while installing sbctl?

Reply:
After Ansible support was added this KW became useless, deleted here 94f9869f37d5ff3a1d278a9fa3372670b959d838 and here 1c1d0f09d55f02794c62f565308cfd0db392fe98.